scss/styles.css

```diff
@@ -0,0 +1,686 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  vertical-align: baseline;
+}
+
+*::before,
+*::after {
+  display: block;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-image: url(../img/tekking.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  min-height: 100vh;
+  font-size: 100%;
+  font-family: "Lato", sans-serif;
+  color: #272a31;
+}
+
+img {
+  max-width: 100%;
+  width: 100%;
+  display: block;
+}
+
+hr {
+  display: inline;
+  position: relative;
+  padding: 0 30px;
+  margin: 1em 0;
+  border-bottom: solid 2px #ec5242;
+}
+
+a {
+  display: block;
+  text-decoration: none;
+  color: inherit;
+  font-size: inherit;
+}
+
+li {
+  list-style-type: none;
+}
+
+h1,
+h2,
+h3,
+p,
+a {
+  font-weight: inherit;
+  font-style: inherit;
+  text-decoration: none;
+  color: inherit;
+}
+
+.header__links {
+  position: fixed;
+  width: 100%;
+  background-color: rgba(211, 211, 211, 0.8470588235);
+  z-index: 10;
+}
+
+@media screen and (min-width: 768px) {
+  .header__links {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-end;
+    align-items: center;
+    position: relative;
+    background-color: rgba(255, 255, 255, 0.9411764706);
+  }
+}
+
+.header__links .hamburger-open {
+  width: 60px;
+  background: none;
+}
+
+@media screen and (min-width: 768px) {
+  .header__links .hamburger-open {
+    display: none;
+  }
+}
+
+.header__links .show-icon {
+  display: none;
+}
+
+.header__links--logo {
+  display: none;
+  position: absolute;
+  z-index: 20;
+  left: 50%;
```

Comment:
I am really impressed with your design skills so far. As stated in the project requirements shown below, you are allowed to personalized the content/text,  however, the pages should use the exact same colors, for text, backgrounds, borders, etc, as the original design. All page elements should also  have similar spacing, borders, shadows, and paddings as the orignal design. 
Kindly check and make sure your design follows this rule. I can see a lot of stylng differences in your project which does not match the original design. A few of them are illustrated below and must be updated. 

- [ ] You have used the border-radius style shown below throughout your project, but it is not seen anywhere in the design. 


### Your design 

![Screenshot from 2023-04-20 10-22-33](https://user-images.githubusercontent.com/65233860/233342468-3fc830af-6424-4c74-9f13-63a57c1d17a7.png)


### Actual design with no border radius

![Screenshot from 2023-04-20 10-22-19](https://user-images.githubusercontent.com/65233860/233342529-9622016f-09b9-41fd-ab69-f79ebb28bfb4.png)